static/templates/template/properties.html

```diff
@@ -23,7 +23,10 @@
     <md-subheader class="pull-left">Data Source</md-subheader>
     <md-input-container>
         <label>Item Source (csv column i.e. label_1)</label>
-        <input ng-model="template.selectedItem.data_source">
+        <!--<input ng-model="template.selectedItem.data_source">-->
```

Comment:
we should show all the available options in the autocomplete when no text is present. Take a look at this for a more native implementation - https://github.com/crowdresearch/crowdsource-platform/blob/develop2/static/js/worker/controllers/workeraccount.controller.js. The skills dropdown uses an autocomplete